### ISSUE TITLE
logs() didn't get entire logs correctly

### ISSUE THREAD

**Issue by ColinHuang** [docker/docker-py#473]
**Using docker logs command**
```
# docker logs 2027436e5ff5e6ef9dcb5ba77df57116c857b7d86d6bc80687fac2c1351e14a2
error: database is uninitialized and MYSQL_ROOT_PASSWORD not set
  Did you forget to add -e MYSQL_ROOT_PASSWORD=... ?
```
**Using docker-py API**
```
atabase is uninitialized and MYSQL_ROOT_PASSWORD not set
  Did you forget to add -e MYSQL_ROOT_PASSWORD=... ?
```

**Comment by shin-** [comment]
Should have been resolved by #336 - What docker-py version are you using?

**Comment by ColinHuang** [comment]
Thanks. It works well after upgrade to the latest version.
